templates/components/common/requireness-msg.html

```diff
@@ -0,0 +1,3 @@
+<small>
+    {{#if required}}{{lang 'common.required'}}{{else}}{{lang 'common.optional'}}{{/if}}
```

Comment:
@bc-as are we sure we want to surface `Optional` as a flag across all not not required fields.

Reply:
@junedkazi Yes, let's add it.  We can re-evaluate this into a page builder toggle if necessary

Reply:
@bc-as can we create a ticket to explore the `Optional` being `Optional` via Page Builder.